src/Donors/Repositories/DonorRepository.php

```diff
@@ -119,6 +120,7 @@ public function insert(Donor $donor)
             'date_created' => Temporal::getFormattedDateTime($dateCreated),
             'user_id' => $donor->userId ?? 0,
             'email' => $donor->email,
+            'phone' => $donor->phone ?? '',
```

Comment:
Since the phone number is optional, I think we want to use the isset approach  here instead so by default it's just an empty column:

```
if (isset($donor->phone)) {
    $args['phone'] = $donor->phone
}
```

Reply:
Resolved in 1002d0d2c812a529db132d79d60538dce2bbca12